code/DDSCodeTester.cpp

```diff
@@ -3969,372 +3969,518 @@ class CustomizedDataReaderListener : public DataReaderListener
 
 void dds_qos_examples()
 {
+    // Taken out of the examples to avoid bloating them
+    DomainParticipantFactory* factory_ = DomainParticipantFactory::get_instance();
+    DomainParticipant* participant_ =
+            DomainParticipantFactory::get_instance()->create_participant(0, PARTICIPANT_QOS_DEFAULT);
+    Subscriber* subscriber_ =
+            participant_->create_subscriber(SUBSCRIBER_QOS_DEFAULT);
+    Publisher* publisher_ =
+            participant_->create_publisher(PUBLISHER_QOS_DEFAULT);
+    Topic* topic_ =
+            participant_->create_topic("TopicName", "DataTypeName", TOPIC_QOS_DEFAULT);
+    DataReader* reader_ = subscriber_->create_datareader(topic_, DATAREADER_QOS_DEFAULT);
+    DataWriter* writer_ = publisher_->create_datawriter(topic_, DATAWRITER_QOS_DEFAULT);
+    uint16_t domain = 7;
     {
         //DDS_CHANGE_DEADLINE_QOS_POLICY
-        DeadlineQosPolicy deadline;
-        //The DeadlineQosPolicy is default constructed with an infinite period.
-        //Change the period to 1 second
-        deadline.period.seconds = 1;
-        deadline.period.nanosec = 0;
+        // This example uses a DataWriter, but it can also be applied to: DataWriter, DataReader and Topic entities
+        DataWriterQos writer_qos;
+        // The DeadlineQosPolicy is default constructed with an infinite period.
```

Comment:
The wording of this sentence is a bit weird to me. The following sounds more natural:
```
// The DeadlineQosPolicy is constructed with an infinite period by default.
```

Reply:
Could we apply the same change to every `is constructed` sentence?